src/main/java/io/oneko/project/rest/ProjectController.java

```diff
@@ -143,8 +143,8 @@ ProjectDTO triggerDeploymentOfVersion(@PathVariable UUID id, @PathVariable UUID
 		WritableProject project = getProjectOr404(id).writable();
 		WritableProjectVersion projectVersion = project.getVersionById(versionId)
 				.orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "Project version with id " + versionId + " not found"));
-		final ReadableProjectVersion deployedVersion = deploymentManager.deploy(projectVersion);
-		return dtoMapper.projectToDTO(deployedVersion.getProject());
+		deploymentManager.deployAsync(projectVersion);
+		return dtoMapper.projectToDTO(projectVersion.getProject().readable());
```

Comment:
Since those deployment requests now only trigger an asynchronous operation that most likely has not been completed by the time we write the response DTO, there probably is no merit in sending a response body in the first place.
So I guess those could be changed to 204-responses and the frontend just doesn't run any in place updated.
I'm not sure, whether or not catnip uses on of those APIs, so we of course should check for compatibility with that as well.